packages/platforms/react-native/__mocks__/react-native-file-access.ts

```diff
@@ -0,0 +1,232 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2020 alpha0010
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * https://github.com/alpha0010/react-native-file-access/blob/7179426e701fa6e54bda6c2a753cfe31a4a08293/LICENSE
+ */
+
+// Copied from v3.0.4:
+// https://github.com/alpha0010/react-native-file-access/blob/7179426e701fa6e54bda6c2a753cfe31a4a08293/jest/react-native-file-access.ts
```

Comment:
is there a reason we copied this rather than install as a dev dependency?

Reply:
Yeah, this file isn't published to NPM: https://www.npmjs.com/package/react-native-file-access?activeTab=code

They encourage you to copy it instead:

> For ease of testing, this library contains a mock implementation: [jest/react-native-file-access.ts](https://github.com/alpha0010/react-native-file-access/blob/master/jest/react-native-file-access.ts). To use, copy it into the [\_\_mocks__](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules) folder, modifying if needed.
> https://github.com/alpha0010/react-native-file-access?tab=readme-ov-file#testing